gwpy/timeseries/timeseries.py

```diff
@@ -1107,7 +1107,7 @@ def filter(self, *filt, **kwargs):
         return new
 
     def transfer_function(self, other, fftlength=None, overlap=None,
-                          window='hann', **kwargs):
+                          window='hann', average='mean', **kwargs):
```

Comment:
This should use the `DEFAULT_FFT_METHOD` module variable to select the default:

```suggestion
                          window='hann', average=DEFAULT_FFT_METHOD, **kwargs):
```

This does change the default to `'median'`, but I think that should be ok.